packages/teleport/src/components/RecoveryCodes/RecoveryCodes.tsx

```diff
@@ -40,6 +42,14 @@ export default function RecoveryCodesDialog({
     });
   };
 
+  const handleContinue = () => {
+    userEventService.capturePreUserEvent({
+      event: CaptureEvent.PreUserOnboardRecoveryCodesContinueClickEvent,
+      username: username,
+    });
+    onContinue();
+  };
```

Comment:
sorry i didn't catch this earlier, components that are defined in `teleport/src/components` means it's shared in multiple places

looks like this use of component will send out an event in the following flows:

1) reset/invite (what our target is)
2) user is logged in -> goes to account settings -> create new recovery codes (`preuser` capture would be incorrect then)
3) going through the recovery flow

not sure if it was okay to be sending out an event for the other cases?

Reply:
Yes, you're right. And I think that is acceptable. If anything, we should rename this action to remove the onboarding prefix since that's misleading. It's a step in onboarding, but not particular to it. I will chat with the team to see if we should rename it now or follow up (since it'll need to change at every layer: webapps, teleport, prehog).

Reply:
Renamed for Clarity:
[prehog](https://github.com/gravitational/prehog/pull/51/files)
[teleport](https://github.com/gravitational/teleport/pull/18142/commits/1335f114146bcb67053b865f70e718606e2ad18b)
[webapps](https://github.com/gravitational/webapps/pull/1344/commits/09c1e887eb7f70dfa10e60d4bebd579ac39f42cd)